resolwe/flow/views/data.py

```diff
@@ -43,6 +43,13 @@ class DuplicateDataSerializer(serializers.Serializer):
     inherit_collection = serializers.BooleanField(default=False)
 
 
+class RestartSerializer(serializers.Serializer):
+    """Serializer for restarting a Data object."""
+
+    contributor = ContributorSerializer(required=False)
```

Comment:
Do we have a use-case where it makes sense to override the contributor?

Reply:
No, removed.